src/lib/dataset/helpers.ts

```diff
@@ -76,7 +76,7 @@ function localUniqueStats(dataset: DataSet) {
  * item's rank is lesser than the other, any positive value otherwise.
  */
 function _statCompare(stat1: ColumnValueStat, stat2: ColumnValueStat) {
-  return stat2.count - stat1.count;
+  return JSON.stringify(stat1.value).localeCompare(JSON.stringify(stat2.value));
```

Comment:
IMO if we have `['Paris', 'berlin', 'paris', 'Berlin']`, we want the output order to be `['Berlin', 'berlin', 'Paris', 'paris']` (or `['berlin', 'Berlin', 'paris',  'Paris']` is also fine, whatever), and not `['Berlin', 'Paris', 'berlin', 'paris']`.

so maybe:

```suggestion
  return JSON.stringify(stat1.value).toLowerCase().localeCompare(JSON.stringify(stat2.value));
```

?

Could you please add a test for this one if you agree ?

Reply:
Aaaah right !! Good point !

Reply:
I added these test to ensure the sort is case insensitive: https://github.com/ToucanToco/weaverbird/pull/532/files#diff-072d2eb768b90ae7ae46f58661967f8aR535